src/drivers/DFRobot_MultiGasSensor.h

```diff
@@ -0,0 +1,326 @@
+/*!
```

Comment:
We don't need this file here. It is already included in the library dependency. Please remove it.

Reply:
Eliminado